quint/src/cliCommands.ts

```diff
@@ -38,109 +43,137 @@ import { Rng, newRng } from './rng'
 import { fileSourceResolver } from './sourceResolver'
 
 export type stage =
-  'loading' | 'parsing' | 'typechecking' |
-  'testing' | 'running' | 'documentation'
+  | 'loading'
+  | 'parsing'
+  | 'typechecking'
+  | 'testing'
+  | 'running'
+  | 'documentation';
 
 /** The data from a ProcedureStage that may be output to --out */
 interface OutputStage {
-  stage: stage,
+  stage: stage;
   // the modules and the lookup table produced by 'parse'
-  modules?: QuintModule[],
-  table?: LookupTable,
+  modules?: QuintModule[];
+  table?: LookupTable;
   // the tables produced by 'typecheck'
-  types?: Map<bigint, TypeScheme>,
-  effects?: Map<bigint, EffectScheme>,
-  modes?: Map<bigint, OpQualifier>,
+  types?: Map<bigint, TypeScheme>;
+  effects?: Map<bigint, EffectScheme>;
+  modes?: Map<bigint, OpQualifier>;
   // Test names output produced by 'test'
-  passed?: string[],
-  failed?: string[],
-  ignored?: string[],
+  passed?: string[];
+  failed?: string[];
+  ignored?: string[];
   // Test names output produced by 'run'
-  status?: 'ok' | 'violation' | 'failure',
-  trace?: QuintEx[],
+  status?: 'ok' | 'violation' | 'failure';
+  trace?: QuintEx[];
   /* Docstrings by defintion name by module name */
-  documentation?: Map<string, Map<string, DocumentationEntry>>,
-  errors?: ErrorMessage[],
-  warnings?: any[], // TODO it doesn't look like this is being used for anything. Should we remove it?
-  sourceCode?: string, // Should not printed, only used in formatting errors
+  documentation?: Map<string, Map<string, DocumentationEntry>>;
+  errors?: ErrorMessage[];
+  warnings?: any[]; // TODO it doesn't look like this is being used for anything. Should we remove it?
+  sourceCode?: string; // Should not printed, only used in formatting errors
```

Comment:
This is a result of running `eslint fix` with our current config, due to defaults:

https://typescript-eslint.io/rules/member-delimiter-style/

But Apparently we aren't supposed to actually be using eslint for this?

I'm a bit confused still on the state our formatting conventions, but just hit a merge conflict due to them, so I will do a cycle to try to get this in order and under CI.